### DISCUSSION TITLE
How do you make website work in iframe

### DISCUSSION THREAD

**Discussion by brand-identity-brand** [DustinBrett/daedalOS#217] [Q&A] [answered]
HI, I am wondering how do you get website iframe working properly for site that doesnt allow it(such as google)? 
I saw this https://github.com/DustinBrett/daedalOS/discussions/138
but it seems like you didnt use BrowserBox. I am guessing you did it purely in frontend? 
I love this project! reminds me of this: https://whimsy.space/
Thanks.

**Comment by DustinBrett** [comment, marked as answer]
Thanks for asking! Specifically for Google I used a special URL they offer that ends with `/webhp?igu=1`. In general it is indeed just a normal iframe and all the bookmarks I've added are ones that support working through the iframe via CORS. I also support IPFS URL's (https://dev.to/dustinbrett/adding-ipfs-to-my-javascript-os-14ag) via IPFS gateways. In the future I hope to add the option to set a proxy on all requests, then it would be possible to access any site and bypass CORS, but it would rely on open/free proxies because I have no desire to host anything on the backend except static files.

**Reply by brand-identity-brand** [reply]
Thanks!
